vm/src/obj/objfloat.rs

```diff
@@ -134,12 +135,16 @@ fn inner_divmod(v1: f64, v2: f64, vm: &VirtualMachine) -> PyResult<(f64, f64)> {
     }
 }
 
-pub fn float_pow(v1: f64, v2: f64, vm: &VirtualMachine) -> PyResult<f64> {
+pub fn float_pow(v1: f64, v2: f64, vm: &VirtualMachine) -> PyResult {
     if v1.is_zero() {
         let msg = format!("{} cannot be raised to a negative power", v1);
         Err(vm.new_zero_division_error(msg))
+    } else if v1.is_sign_negative() && v2.floor() != v2 {
```

Comment:
Check the clippy error please. In local, `cargo clippy --all` command works. We also can see the result in CI result too.

Reply:
Updated PR. Thanks 👍